src/flake.rs

```diff
@@ -15,9 +15,12 @@ use parse_flake_lock::{FlakeLock, Node};
 pub const ALLOWED_REFS: &[&str] = &[
     "nixos-23.05",
     "nixos-23.05-small",
+    "nixos-23.11",
+    "nixos-23.11-small",
     "nixos-unstable",
     "nixos-unstable-small",
     "nixpkgs-23.05-darwin",
+    "nixpkgs-23.11-darwin",
```

Comment:
Wouldn't it better to just assume anything matching the pattern with any digits is supported instead of updating this on schedule every half a year?

Reply:
No, because only the two most recent are considered supported. A branch like `nixpkgs-20.09` is considered deprecated.